shiny/_main.py

```diff
@@ -41,6 +41,7 @@ def main() -> None:
     "*.htm",
     "*.html",
     "*.png",
+    "_brand*.yml",
```

Comment:
```suggestion
    "_brand*.yml",
    "_brand*.yaml",
```

Reply:
I guess? And come to think of it, maybe just `*.yml` and `*.yaml`?

Reply:
Agreed